Training 45/pokemon.js

```diff
@@ -0,0 +1,38 @@
+
+function Pokemon(p_name, p_type, p_attack, p_defense){
```

Comment:
please use POO in javascripts. These functions damage, fight, ... can go in a js class